fiftyone/utils/labelbox.py

```diff
@@ -641,8 +644,13 @@ def download_annotations(self, results):
 
         project = self._client.get_project(project_id)
         labels_json = self._download_project_labels(project=project)
-        is_video = results._samples.media_type == fomm.VIDEO
-
+        is_video = (results._samples.media_type == fomm.VIDEO) or (
+            results._samples.media_type == fomm.GROUP
+            and results._samples.group_media_types[
```

Comment:
I think `results._samples.group_media_types` will be `None` for dynamic groups. But `_parent_media_type` will be defined.

Not sure if dynamic groups should be supported. There is also dynamically grouped groups dataset 😅

Reply:
Good catch! I should have been more clear, but annotating any kind of grouped datasets themselves is NOT supported today: https://github.com/voxel51/fiftyone/blob/21ff9a49a93115210cd5e4d34a8653c3ad0b3c24/fiftyone/utils/labelbox.py#L113

This is specifically if a slice has been selected from a group dataset that has had `select_group_slices()` called on it, or some other way to get an Image/Video view out of the group.

Reply:
Ok, and this happens only on download 👍  💂